### PR TITLE
Fix submission comment animation

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-13465
affects: student
release note: Fixed a bug when viewing submission comment attachments

Test plan:
* Student App > narmstrong > s > Dashboard > CS 1400 > Assignments > Annotate this > Submission & Rubric > Comments > jpg attachment
* Modal animation should not be transparent
* Go back to Dashboard > Drawer > Files
* Tap around a few different file types to makes sure file viewer is all good

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1009952&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
#### Release Note: 
Fixed a bug when viewing submission comment attachments

#### Affected Apps: Student

[MBL-13465](https://instructure.atlassian.net/browse/MBL-13465)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.23% | 93.24% | -0.01%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/TechDebt/Files and Folders/FileViewController.m [hunk 1]
```diff
@@ -98,7 +98,7 @@ - (void)viewDidLoad {
 
     [self updateForURLState];
     
-    [self.view setBackgroundColor:[UIColor colorNamed:@"backgroundLightest" inBundle:NSBundle.core compatibleWithTraitCollection:nil]];
```
Comment (armstrongnate) [inline Student/TechDebt/Files and Folders/FileViewController.m]: This was returning `nil`. I couldn't figure out how to get the color using `colorNamed:inBundle:` so I added an `@objc` helper to Core.

File: scripts/build-colors.js [hunk 2]
```diff
@@ -228,6 +228,11 @@ extension UIColor {
         let named = inHighContrast ? "\\(name.rawValue)HighContrast" : name.rawValue
         return UIColor(named: named, in: .core, compatibleWith: nil)!
     }
+
+    @objc public static func named(_ name: String) -> UIColor? {
+        guard let color = InstColorName(rawValue: name) else { return nil }
+        return named(color)
+    }
```
Comment (vanwagonet) [inline scripts/build-colors.js]: good job adding the helper to the script so it doesn't get lost the next time it is run.

**Pull request merged**